server/app/views/admin/programs/ProgramStatusesView.java

```diff
@@ -334,8 +348,38 @@ private Modal makeStatusUpdateModal(
     Messages messages = messagesApi.preferred(request);
     ProgramStatusesForm formData = form.value().get();
 
+    Boolean isCurrentDefault = formData.getDefaultStatus().orElse(false);
+    LabelTag defaultCheckbox =
+        label()
+            .with(
+                input()
+                    .withType("checkbox")
+                    .withName(ProgramStatusesForm.DEFAULT_CHECKBOX_NAME)
+                    .withCondChecked(isCurrentDefault)
+                    .withClasses(
+                        BaseStyles.CHECKBOX,
+                        "default-checkbox-" + formData.getConfiguredStatusText()),
+                span("Set as default status"));
+    Optional<StatusDefinitions.Status> currentDefaultStatus =
+        program.toProgram().getDefaultStatus();
+    String defaultStatusText =
+        currentDefaultStatus.isPresent() ? currentDefaultStatus.get().statusText() : "";
+    String messagePart =
+        currentDefaultStatus.isPresent() ? "from " + defaultStatusText + " to " : "to ";
+    // I'm so sorry
```

Comment:
Heh, I should probably remove this comment.  I really don't love this at all.  But it seemed like all the other options required twisting the code in knots to show a nice confirmation modal.  If anyone has ideas on how to do this better, please let me know!